docs/Getting Started/NixOS/Root on ZFS/2-system-installation.rst

```diff
@@ -0,0 +1,132 @@
+.. highlight:: sh
```

Comment:
Minor note for future - file rename will lead to different link, if it's possible - we should leave same links to minimize 404s on external links to our docs.